### DISCUSSION TITLE
Autosave drafts for issues & comments

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [community/community#39676] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi, I write a fair lot of issues here on GitHub in all sorts of repositories. Sometimes I have to provide longer explanations (like suggest ways of implementing a feature or going into nitty gritty detail of how a new feature should operate, or how to reproduce rare bugs) and its not rare for me to end up spending an hour or 2 writing them, perhaps taking a break somewhere in the middle - likewise its not rare for my web browser to crash every once in a while, or for me to accidentally close the wrong browser window, or for MS Windows to freak out.

Could we see a feature to store drafts of issues & comments so that when you navigate back to a page after leaving it, there will be an option to reveal unposted previous drafts on the page you are writing (perhaps embedded directly in Githubs text editor)?

**Comment by david-engelmann** [comment]
I've noticed decent auto saving of issues and comments into draft when I've accidentally existed out of a page I was working on. It doesn't work in Incognito mode or if you've clear cookies.

**Comment by Alex0vSky** [comment]
I just lost the changes to my README.md.
Sorry, I didn't know with whom to share my sadness.

Just "goback" in browser in "Preview" step

**Comment by s-harbage** [comment]
I completely agree—this feels essential for a modern collaboration platform.
Even JIRA and other project-management tools already autosave drafts, so it’s surprising that GitHub doesn’t.
Losing carefully written issue/draft descriptions or comments because of a browser crash, accidental tab close, or flaky network is both frustrating and discouraging for contributors.
An autosave / draft-recovery feature would prevent lost work and make GitHub more welcoming to open-source maintainers and occasional contributors alike.

Please up-vote if you agree!
